src/components/skills-quiz-v2/styles/_Body.scss

```diff
@@ -0,0 +1,37 @@
+$gray-500: #707070;
+$max-sm-modal-width: 28rem !important;
+$min-sm-modal-height: 16rem !important;
+
+.page-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.form {
+  width: 65%;
+}
+.pgn__form-group {
+  width: 101%;
+}
+.text {
+  width: 65%;
+}
+.pgn__form-label {
+  font-weight: bold;
+}
+
+.btn {
+  margin: 7px -5px;
+}
```

Comment:
To get around the issue of having styles apply globally across the entire MFE application is to nest generic style overrides in a class name that is specific to the feature/page route/etc. you're hoping the styles should apply to.

For example, add a class name to the top-most element of skills quiz v2 with a class name of `skills-quiz-v2` and then have all these custom styles nested under the `.skills-quiz-v2` class name so that the generic styles don't impact anything outside of `.skills-quiz-v2`.